.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: feature request
+assignees: ''
+---
+
+**Description:**
+Describe your proposal.
```

Comment:
It seems to be a too simple template.
I think there should be fields like:
- Proposal overview
- Use case or justification
- Ask for willing to submit PR if we accept customers contributions

Reply:
+1